libs/island-ui/core/src/lib/Button/Button.tsx

```diff
@@ -64,6 +64,7 @@ export const Button = forwardRef<
       white,
       tabIndex,
       rounded = false,
+      ...rest
```

Comment:
we should not have the button component spread like this, we are rewriting the button component so this will change